app-modules/authorization/src/Http/Controllers/SocialiteController.php

```diff
@@ -36,7 +36,7 @@ public function callback(SocialiteProvider $provider)
             ->where('email', $socialiteUser->getEmail())
             ->first();
 
-        if (! $user?->is_external) {
+        if ($user?->is_external) {
```

Comment:
I think the `!` needs to stay doesn't it?

`is_external` represents that their login is external, i.e. they log in via SSO.

Reply:
Ahh yeah I think I misunderstood that field, and the nature of the external login. I thought the external abilities were for vendors who might not be part of an organization, and therefore may not be able to authenticate through OAuth?

Reply:
Yeah. Do you think there is a case for renaming `is_external`?

Reply:
Confirmed via external conversation - added a ticket to the backlog to address this.